varnish/plan.sh

```diff
@@ -9,10 +9,31 @@ pkg_version="5.1.2"
 pkg_source="https://varnish-cache.org/_downloads/${pkg_name}-${pkg_version}.tgz"
 
 pkg_shasum="39d858137e26948a7c85f07363f13f0778da61d234126e03a160a0cb9ba4fce3"
-pkg_deps=(core/glibc core/ncurses core/docutils core/pcre core/gcc core/bash )
-pkg_build_deps=(core/make core/python2 core/pkg-config core/readline core/graphviz core/libtool core/libedit core/automake core/m4 core/autoconf)
+pkg_deps=(
+  core/bash
+  core/gcc
```

Comment:
Can this change to gcc-libs?

Reply:
I'll give it a shot.  What's the difference?

Reply:
`core/gcc` brings the whole compiler along. `gcc-libs` is just the things the compiler might link to.

Reply:
ahhhhhhh!!!  Makes total sense.  Thanks!

Reply:
Yup, that should do it!